web/styles/compose.css

```diff
@@ -770,14 +770,6 @@ a.compose_control_button {
     }
 }
 
-/* This is used to override the
- * properties of `a.compose_control_button`
- * without using `!important`.
- */
-a.compose_control_button.hide {
-    display: none;
-}
-
```

Comment:
Based on the comment here, the key issue was likely that previously there was a `display:` settings on `a.compose_control_button`, which would have taken precedence over the `.hide` class rule.